cssdb.json

```diff
@@ -586,7 +586,7 @@
         "link": "https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-exponential-functions"
       }
     ],
-    "vendors_implementations": 2
+    "vendors_implementations": 1
```

Comment:
This would be breaking if released now, if we wait until the firefox 118 release (end of september) it would be non-breaking.